vspace/src/main/webapp/WEB-INF/views/staff/spaces/space.jsp

```diff
@@ -187,6 +245,54 @@ $( document ).ready(function() {
 		});
 	});
 	
+	$("#createModuleLinkBtn").click(function(e) {
+		e.preventDefault();
+		
+		if (storeX == undefined || storeY == undefined) {
+			$("#errorMsg").text("Please click on the image to specify where the new link should be located.")
+			$('#errorAlert').show();
+			return;
+		}
+		
+		var linkedModules = $("#linkedModule").val();
+		console.log(linkedModules);
+		
+		if (linkedModules == undefined || linkedModules == "") {
+			$("#errorMsg").text("Please select linked modules.")
+			$('#errorAlert').show();
+			return;
+		}
+		
+		$("#moduleLinkX").val(storeX);
+		$("#moduleLinkY").val(storeY);
+		
+		var form = $("#createModuleLinkForm");
+		var formData = new FormData(form[0]);
+		
+		var moduleLinkInfo = createModuleLinkInfo();
+        
+	    $.ajax({
+			type: "POST",
+			url: "<c:url value="/staff/space/${space.id}/modulelink?${_csrf.parameterName}=${_csrf.token}" />",
+			cache       : false,
+	        contentType : false,
+	        processData : false,
+	        enctype: 'multipart/form-data',
+	        data: formData, 
+	        success: function(data) {
+	        	var linkData = JSON.parse(data);
+	        	$("#bgImage").off("click");
+	        	moduleLinkInfo["id"] = linkData["id"];
+	            showModuleLink(moduleLinkInfo, true);
+	            $("#module_label").attr("id","");
+	            $("#link").attr("id","");
+	            $("#createModuleLinkAlert").hide();  
+	            $("#errorMsg").text("")
+	            $('#errorAlert').hide();
+	        }
```

Comment:
indentation

Reply:
could you please clarify whats wrong here. In my eclipse indentation seems fine.